addons/medical_feedback/functions/fnc_handleEffects.sqf

```diff
@@ -42,8 +43,17 @@ if ((!GVAR(heartBeatEffectRunning)) && {_heartRate != 0} && {(_heartRate > 160)
 // - Visual effects -----------------------------------------------------------
 [_unconscious, 2] call FUNC(effectUnconscious);
 [
-    true, linearConversion [BLOOD_VOLUME_CLASS_2_HEMORRHAGE, BLOOD_VOLUME_CLASS_4_HEMORRHAGE, _bloodVolume, 0, 1, true]
+    true,
+    linearConversion [BLOOD_VOLUME_CLASS_2_HEMORRHAGE, BLOOD_VOLUME_CLASS_4_HEMORRHAGE, _bloodVolume, 0, 1, true]
 ] call FUNC(effectBloodVolume);
+[
+    true,
+    ceil(linearConversion [
```

Comment:
```suggestion
    ceil linearConversion [
```
then delete the rogue closing parentheses as well.

Reply:
Fixed